app/src/main/res/values/strings.xml

```diff
@@ -78,6 +78,7 @@
 
     <!--ticket details-->
     <string name="your_info">Your Info</string>
+    <string name="more_attendee_info">More Attendee Information</string>
```

Comment:
use something more appropriate. see what Evenrbrite uses

Reply:
Changed it to `Other Information`